pkg/pb/utils.go

```diff
@@ -63,15 +62,6 @@ func CopyNode(src *Node) (dst *Node) {
 	return &node
 }
 
-// DPanicOnInvalid panics if NodeType is invalid if zap is in development mode,
```

Comment:
Shouldn't this only panic when it's in dev mode?

Reply:
Yes, but the rest of the code is invalid if it doesn't panic, the error should be properly handled. Either way, removing this completely so people won't accidentally use this for production things.